integration/test/__main__.py

```diff
@@ -56,11 +52,16 @@
 from test_launch_pthread import *
 from test_geopmagent import *
 from test_environment import *
-from test_power_governor import *
 from test_frequency_map import *
 from test_hint_time import *
 from test_progress import *
 
+if 'GEOPM_RUN_LONG_TESTS' in os.environ
```

Comment:
missing ```:``` at EOL

Was this tested?

Reply:
Obviously not :( thanks, will run main...